UmpleToJava/UmpleTLTemplates/toJson_declare.ump

```diff
@@ -180,17 +190,7 @@ class UmpleToJava {
 
   !>>  
   
-  toJson_declare <<!<</*toJson_declare*/>>
-  /*
-  * Generate Json for this object and connected objects visited objects to enable avoidance of infinite loops
-  *
-  * @param toJsonOutput  Output is aded to this as the network of objects is traversed
-  * @param visitedList  Every concrete object visited is added so we don't re-outpu
-  * @param nestLevel    As we output deeper objects, indent them more
-  * @param atConcreteClass false when we are recursing to a superclass
-  *     so we get the superclass data
-  * @return whether or not anything was output (so we can tell whether we need to output a comma)
-  */  
+  toJson_declare <<!<</*toJson_declare*/>> 
```

Comment:
Why did you remove this block of comments for the toJson?

Reply:
I moved this comment to the very top of the class, because the @return statement explains what toJsonHelper returns.

Reply:
Ah I see. It was in the wrong place! ... but there should be a comment generated before toJson (although it would not have any param descriuptions)

Reply:
For sure, I just added a comment before toJson describe it would return a string in Json format